standard-cluster-services/src/main/java/org/terracotta/monitoring/IMonitoringProducer.java

```diff
@@ -0,0 +1,31 @@
+package org.terracotta.monitoring;
+
+import com.tc.classloader.CommonComponent;
+
+
```

Comment:
I assume that we will just augment this interface.

It will really depend on what that looks like and where we want to go with it.  I would like to avoid having to hold multiple resolved service interfaces in the consumer if we expect that both stats and events would be desired.